scss/_regions/_container.scss

```diff
@@ -59,6 +60,8 @@
       }
 
       &.-half {
+        line-height: $unitless-line-height; // reset to "base" line height for smaller columns
```

Comment:
Does the comment mean you are gonna update the name of this variable back to "base"? `comfortable-line-height` feels a little generic to me, like it's not clear when you are supposed to use it.